### DISCUSSION TITLE
How to Use Weblate Client (wlc) with Categories and Understanding --config-section Argument

### DISCUSSION THREAD

**Discussion by vanlinh2204** [WeblateOrg/weblate#10399] [Q&A]
ello Weblate team,

I'm facing an issue while using the Weblate client (wlc) with categories. Here's a brief overview of my configuration:

```
[weblate]
url = http://localhost:8080/api/
key = wlu_CePzAJOWKoFEDI9k3bf1bZcEeeMApYTcjJ47
translation = ipension/test_component
```
When I run the wlc command with the override config file option:

```
wlc --config ./weblate.ini show
```
Everything works as expected. However, when I move my component to a category, modifying the configuration to:

```
[weblate]
url = http://localhost:8080/api/
key = wlu_CePzAJOWKoFEDI9k3bf1bZcEeeMApYTcjJ47
translation = ipension/test_category/test_component
```
and run the wlc command again:

```
wlc --config ./weblate.ini show
```
I encounter the following exception:

```
Error: Object not found on the server (maybe the operation is not supported on the server).
```

My question is: Am I passing the path to the translation component correctly, or does wlc not support categories?

Additionally, I'm curious about how to use the --config-section argument. Could you provide an example?

Thank you for your assistance!

**Comment by nijel** [comment]
The category needs to be URL encoded in API, see https://docs.weblate.org/en/latest/api.html#components-and-categories. Seems like wlc documentation was not updated to cover this…

So it should be `ipension/test_category%2Ftest_component`.
